### PR TITLE
Add tabs to System Information page

### PR DESCRIPTION
**Pull request by janfaracik**
Small PR to add tabs to the System Information page. This makes it a little easier to navigate as previously it required lots of scrolling to get to any section that wasn't System Properties.

Memory Usage also now fits correctly on the page, you can change the timespans without having to reload the page and the image is now only loaded if the tab is clicked.

**Before**
image

image

**After**
image

image

### Testing done

* Open up the System Information page, check that existing sections are still present and that each section has a tab.
* For the Memory Usage tab check that cycling through the select works and that the image loaded is correct.

### Proposed changelog entries

- Add tabs to System Information page

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@jenkinsci/sig-ux 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7373"></a>

### CODE REVIEW

**Review by yaroslavafenkin — APPROVED** [review]
Seems fine from security POV

File: core/src/main/resources/jenkins/model/Jenkins/systemInfo.jelly
Comment (daniel-beck) [inline core/src/main/resources/jenkins/model/Jenkins/systemInfo.jelly:97]: I always understood timespan to refer to the total time displayed in the graph (6 hrs, 20 hrs, and a month or so?), rather than the update interval/granularity. Was this wrong? If not, do the new labels still make sense?

File: core/src/main/resources/jenkins/model/Jenkins/systemInfo.jelly
Comment (daniel-beck) [inline core/src/main/resources/jenkins/model/Jenkins/systemInfo.jelly:38]: Should be a set of Jelly tags rather than forcing people to copy & paste internal classes around.

**Review by daniel-beck — COMMENTED** [review]
Looks reasonable overall. Switching between timespans works a lot better than before on this particular page, even if it takes more clicks, but introduces a UI inconsistency with `load-statistics` pages on nodes and labels.

**Comment by janfaracik** [comment]
> but introduces a UI inconsistency with `load-statistics` pages on nodes and labels.

For the load statistics page, could that be bundled into this page to have one place for stats?

**Comment by daniel-beck** [comment]
> For the load statistics page, could that be bundled into this page to have one place for stats?

There's an infinite number of load-statistics pages (one global, one per node, one per label), so, no.

File: core/src/main/resources/jenkins/model/Jenkins/systemInfo.jelly
Comment (janfaracik) [reply core/src/main/resources/jenkins/model/Jenkins/systemInfo.jelly:100]: Hmm I'm not too sure actually. I've restored the original labels for the time being, would anybody from @jenkinsci/sig-ux know the answer to this?

**Review by timja — APPROVED** [review]

File: core/src/main/resources/lib/layout/tabPane.jelly
Comment (NotMyFault) [inline core/src/main/resources/lib/layout/tabPane.jelly:30]:
```suggestion
    </st:attribute>
    @since TODO
```

**Review by NotMyFault — APPROVED** [review]

**Comment by timja** [comment]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Comment by daniel-beck** [comment]
Caused [JENKINS-75692](https://issues.jenkins.io/browse/JENKINS-75692).

**Pull request merged**
